core/src/main/java/io/confluent/rest/Application.java

```diff
@@ -245,6 +244,14 @@ protected void doStop() throws Exception {
       context.setBaseResource(staticResources);
     }
 
+    if(config.getBoolean(RestConfig.ENABLE_GZIP_COMPRESSION_CONFIG)) {
+      FilterHolder gzipFilter = new FilterHolder(AsyncGzipFilter.class);
+      // do not check if .gz file already exists for the requested resource
+      gzipFilter.setInitParameter("checkGzExists", "false");
+      gzipFilter.setInitParameter("methods", "GET,POST");
+      context.addFilter(gzipFilter, "/*", null);
```

Comment:
Should there be a way to select what resources to apply compression to. More specifically, would it make sense for the config to be the resources to apply it to rather than a boolean?